src/components/common/ColonyActions/ActionDetailsPage/DefaultMotion/MotionPhaseWidget/StakingWidget/StakingInput/StakingControls/ObjectButton.tsx

```diff
@@ -1,31 +1,19 @@
 import React from 'react';
 
-import RaiseObjectionDialog from '~common/Dialogs/RaiseObjectionDialog';
 import Button from '~shared/Button';
-import { useDialog } from '~shared/Dialog';
-
-import { useStakingWidgetContext } from '../../StakingWidgetProvider';
+import { useObjectButton } from '~hooks';
```

Comment:
I'm curious why you dont place the useObjectButton hook this in the same folder with ObjectButton.tsx? Similar to what you recommended for the useColonyMembers hook.

Reply:
i think that's a good pattern, just not one I've been following so far in this project. I will create an issue to do it everywhere

Reply:
https://github.com/JoinColony/colonyCDapp/issues/354